RF24Network.cpp

```diff
@@ -1234,8 +1233,7 @@ uint64_t ESBNetwork<radio_t>::pipe_address(uint16_t node, uint8_t pipe)
     #endif
 #endif
     };
-    uint64_t result = 0xCCCCCCCCCCLL;
-    uint8_t* out = reinterpret_cast<uint8_t*>(&result);
+    uint8_t* out = address;
```

Comment:
If we added a memset(address, 0xCC, 5); here, then you could just pass a blank array to this function